renovate.json

```diff
@@ -10,9 +10,18 @@
     "dependencies"
   ],
   "postUpdateOptions": [
-    "gomodTidy",
     "gomodUpdateImportPaths"
   ],
+  "postUpgradeTasks": {
+    "commands": [
+      "for mod in $(go list -f '{{.Dir}}' -m); do (cd $mod; go mod tidy); done"
+    ],
+    "fileFilters": [
+      "**/go.mod",
+      "**/go.sum"
+    ],
+    "executionMode": "update"
```

Comment:
```suggestion
```
update is default so we can also omit. omitting does not make the config less clear to me.